From 98f6cac7054cd240cde28dc237168efbd35747c2 Mon Sep 17 00:00:00 2001
From: "Hebbar, Gururaja" <gururaja.hebbar@ti.com>
Date: Tue, 8 May 2012 12:17:19 +0530
Subject: [PATCH 0948/1092] ARM: OMAP2+: am335x: rename i2c boardinfo struct
 for clarity

rename all i2c boardinfo structures based on the i2c bus interface.

	am335x_i2c_boardinfo  --> am335x_i2c0_boardinfo
	am335x_i2c_boardinfo1 --> am335x_i2c1_boardinfo
	am335x_i2c_boardinfo2 --> am335x_i2c2_boardinfo

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index e43cdba..59713f6 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -1362,7 +1362,7 @@ static struct lis3lv02d_platform_data lis331dlh_pdata = {
 	.st_max_limits[2] = 750,
 };
 
-static struct i2c_board_info am335x_i2c_boardinfo1[] = {
+static struct i2c_board_info am335x_i2c1_boardinfo[] = {
 	{
 		I2C_BOARD_INFO("tlv320aic3x", 0x1b),
 	},
@@ -1381,20 +1381,20 @@ static struct i2c_board_info am335x_i2c_boardinfo1[] = {
 static void i2c1_init(int evm_id, int profile)
 {
 	setup_pin_mux(i2c1_pin_mux);
-	omap_register_i2c_bus(2, 100, am335x_i2c_boardinfo1,
-			ARRAY_SIZE(am335x_i2c_boardinfo1));
+	omap_register_i2c_bus(2, 100, am335x_i2c1_boardinfo,
+			ARRAY_SIZE(am335x_i2c1_boardinfo));
 	return;
 }
 
 
-static struct i2c_board_info am335x_i2c_boardinfo2[] = {
+static struct i2c_board_info am335x_i2c2_boardinfo[] = {
 };
 
 static void i2c2_init(int evm_id, int profile)
 {
 	setup_pin_mux(i2c2_pin_mux);
-	omap_register_i2c_bus(3, 100, am335x_i2c_boardinfo2,
-			ARRAY_SIZE(am335x_i2c_boardinfo2));
+	omap_register_i2c_bus(3, 100, am335x_i2c2_boardinfo,
+			ARRAY_SIZE(am335x_i2c2_boardinfo));
 	return;
 }
 
@@ -1996,7 +1996,7 @@ static struct tps65910_board am335x_tps65910_info = {
 *	   the below struct. Daughter boards eeprom are probed 1st. Baseboard
 *	   eeprom probe is called last.
 */
-static struct i2c_board_info __initdata am335x_i2c_boardinfo[] = {
+static struct i2c_board_info __initdata am335x_i2c0_boardinfo[] = {
 	{
 		/* Daughter Board EEPROM */
 		I2C_BOARD_INFO("24c256", DAUG_BOARD_I2C_ADDR),
@@ -2070,8 +2070,8 @@ static void __init am335x_evm_i2c_init(void)
 
 	evm_init_cpld();
 
-	omap_register_i2c_bus(1, 100, am335x_i2c_boardinfo,
-				ARRAY_SIZE(am335x_i2c_boardinfo));
+	omap_register_i2c_bus(1, 100, am335x_i2c0_boardinfo,
+				ARRAY_SIZE(am335x_i2c0_boardinfo));
 }
 
 static struct resource am335x_rtc_resources[] = {
-- 
1.7.11.2

